### ISSUE TITLE
Not able to login from mobile SDK

### ISSUE THREAD

**Issue by dishasyno** [mapillary/mapillary-sdk-ios#5]
Hello,

I have downloaded the mapillary demo from GIT. I have already created an account on mapillary but when I try to login with the same credentials through mobile it gives me an error.
Kindly help asap.

Thank you.

**Comment by millenbop** [comment]
Hi!

Have you created a new client id for the app?

**Comment by dishasyno** [comment]
Yes, In the developer's section of mapillary I can see my app and client ID for the app too. And the callback url is my appPackageName.mapillary . Is this the correct way ?
